src/witan/send/adroddiad/ncy_analysis.clj

```diff
@@ -0,0 +1,274 @@
+(ns witan.send.adroddiad.ncy-analysis
+  (:require [cljplot.build :as plotb]
+            [cljplot.config :as cfg]
+            [cljplot.render :as plotr]
+            [kixi.large :as large]
+            [kixi.plot :as plot]
+            [kixi.plot.colors :as colors]
+            [kixi.plot.series :as series]
+            [tablecloth.api :as tc]
+            [tech.v3.datatype.functional :as dfn]
+            [witan.send.adroddiad.chart-utils :as chart-utils]
+            [witan.send.adroddiad.simulated-transition-counts :as stc]
+            [witan.send.adroddiad.summary :as summary]
+            [witan.send.adroddiad.transitions :as tr]))
+
+(defn delta-column [ds column-name]
+  (tc/map-columns ds column-name [(name column-name) (str "[:calendar-year]-aggregation." (name column-name))] (fn [a b] (- a b))))
+
+(defn fill-in-years [ds years-1 years-2]
+  (reduce #(tc/concat-copying %1 (tc/dataset {:low-95 0 :min 0 :q1 0 :q3 0 :median 0
+                                              :max 0 :row-count 1 :high-95 0 :calendar-year %2}))
```

Comment:
Pending addressing the incomplete `:simulation :calendar-year` issue, suggest set `:row-count` to `0` (rather than `1`), so consistent.